arm/Microsoft.ServiceFabric/clusters/parameters/parameters.json

```diff
@@ -0,0 +1,238 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "serviceFabricClusterName": {
+            "value": "sxx-az-sfc-weu-x-666"
+        },
+        "location": {
+            "value": "centralIndia"
+        },
+        "tags": {
+            "value": {
+                "resourceType": "Service Fabric",
+                "clusterName": "sxx-az-sfc-weu-x-666"
+            }
+        },
+        // "lock": {
+        //     "value": "CanNotDelete"
+        // },
+        "addOnFeatures": {
+            "value": ["DnsService", "RepairManager"]
+        },
+        // "maxUnusedVersionsToKeep": {
+        //     "value": 2
+        // },
+        // "azureActiveDirectory": {
+        //     "value": {
+        //         "clientApplication": "",
+        //         "clusterApplication": "",
+        //         "tenantId": ""
+        //     }
+        // },
+        "certificate": {
+            "value": {
+                "thumbprint": "154813CEE741AE4FC28CAE17FC16FB30888FACF3", // If 'certificate' is provided then this is mandatory
+                //"thumbprintSecondary": "",
+                "x509StoreName": "My" // Possible values are 'AddressBook', 'AuthRoot', 'CertificateAuthority', 'Disallowed', 'My', 'Root', 'TrustedPeople', 'TrustedPublisher'
+            }
+        },
+        // "certificateCommonNames": {
```

Comment:
Ideally we should not have comments in here. If we want examples for services, please move them into the readme's resource usage section

Reply:
Removed.

Reply:
Note that the current set of parameters have been tested.
Some of the additional parameters like _serviceFabricClusterApplications_ and _serviceFabricClusterApplicationTypeshave_ been intentionally commented, so that, they remain as reference to the audience. Their actual use will arise when VMSS behind the SFC has been implemented.